spyderlib/utils/icon_manager.py

```diff
@@ -1,8 +1,11 @@
 import os
```

Comment:
Copyright assignment is missing in this file. Please add it before we forget about it.